test/plug_test.exs

```diff
@@ -649,4 +660,144 @@ defmodule PlugTest do
       end
     end
   end
+
+  defmodule Preload do
+    use ExUnit.Case, async: true
+  
+    test "it loads the load resource correctly when the :preload key is specified" do
```

Comment:
Just a small grammatical error here :).

Should be "it loads the resource correctly when the :preload key is specified"